windows_prepare_python.bat

```diff
@@ -49,23 +61,23 @@ if exist %dest_dir%\%python_full_name%.zip (
     if !errorlevel! neq 0 exit /b !errorlevel!
 )
 
-tar -xf %dest_dir%\%python_full_name%.zip -C %dest_dir%\%python_full_name%
+C:\Windows\System32\tar.exe -xf %dest_dir%\%python_full_name%.zip -C %dest_dir%\%python_full_name%
 if !errorlevel! neq 0 exit /b !errorlevel!
 
 cd %dest_dir%\%python_full_name%
-md python39
+md %python_dir%
```

Comment:
Name `python_dir` doesn't fit that anymore since it's used for `._pth` file. 
Maybe we could go with something like `python_short_name` or `python_version_short`?